core/src/main/java/com/linecorp/armeria/internal/common/Http1KeepAliveHandler.java

```diff
@@ -24,7 +24,7 @@ protected Http1KeepAliveHandler(Channel channel, String name, Timer keepAliveTim
                                     long pingIntervalMillis, long maxConnectionAgeMillis,
                                     long maxNumRequestsPerConnection) {
         super(channel, name, keepAliveTimer, idleTimeoutMillis, pingIntervalMillis, maxConnectionAgeMillis,
-              maxNumRequestsPerConnection);
+              maxNumRequestsPerConnection, false);
```

Comment:
Just double-checking, there is no `PING` frame in `HTTP/1` so we should only add this option to `Http2KeepAliveHandler`, right?

cc: @trustin @ikhoon

Reply:
It is yes or no. We use `OPTIONS *` as PING requests for HTTP/1 clients although there is no PING in HTTP/1 servers.
https://github.com/line/armeria/blob/92e70170b560470a68ae18895d60b3689c8d98bb/core/src/main/java/com/linecorp/armeria/client/Http1ClientKeepAliveHandler.java#L59-L70

Reply:
But there is no keep-alive connection in HTTP/1 so it should be false by default, right?

Reply:
In Armeria, HTTP/1 implies HTTP/1.1 in which Keep-Alive option is enabled by default.
https://www.rfc-editor.org/rfc/rfc9112#name-persistence
Please correct me if I'm mistaken.

Reply:
So what should be the final change here 😅

Reply:
@ikhoon Thoughts? 😅 Should we cound `OPTIONS *` as a ping?

Reply:
I will resolve all comments once we settle down in this 🙂

Reply:
> Should we cound OPTIONS * as a ping?

https://github.com/line/armeria/blob/5f6a3b4836a9f0e15d4d074729054993597888c8/core/src/main/java/com/linecorp/armeria/common/Flags.java#L794-L798

I have no objection to this, as long as it is clearly limited to applying `keepAliveOnPing` to HTTP/2. But since we're already using `OPTION *` as a ping, why not apply it to HTTP/1?

Reply:
OK, I will get back to this tomorrow evening 👍

Reply:
Oops, I missed this thread. 😓 
Sorry about the wrong answer and I agree that we should count the options requests as pings.